src/raw/accessor.rs

```diff
@@ -476,6 +480,17 @@ impl AccessorInfo {
         self
     }
 
+    /// Batch size limit of backend, only meaningful when the backend supports batch operation
+    pub fn batch_limit(&self) -> usize {
```

Comment:
Please reserve this API as `pub(crate)`, I'm not ready to make it available to our users.